cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: node-labeler
+resources:
+- namespace.yaml
```

Comment:
the namespace is in base now.

Reply:
...so it should probably be removed from here.

Reply:
Duh, never mind. Sorry!

Reply:
I did delete `node-labeler/namespace.yaml`, this is `cluster-scope/base/core/namespaces/node-labeler/kustomization.yaml`